test-cypress/support/repository-commands.js

```diff
@@ -0,0 +1,22 @@
+import repoTemplate from '../fixtures/repo-template.json';
+import _ from 'lodash';
```

Comment:
Uhmm actually Cypress wraps the lodash library already. Try `Cypress._.defaultsDeep`

Reply:
Huh, ok. Will use it.